Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -230,6 +230,12 @@ private async Task ErrorHandler(Exception ex, IRequest httpReq, int statusCode,
 
             httpRes.StatusCode = statusCode;
 
+            if (!_hostEnvironment.IsDevelopment())
+            {
+                await httpRes.WriteAsync("Error processing request.").ConfigureAwait(false);
+                return;
+            }
+
             var errContent = NormalizeExceptionMessage(ex) ?? string.Empty;
```

Comment:
```suggestion
            var errContent = _hostEnvironment.IsDevelopment()
                    ? (NormalizeExceptionMessage(ex) ?? string.Empty)
                    : "Error processing request.";
```
otherwise you're losing `.ContentType` and `.ContentLength` fields